services/submission.js

```diff
@@ -295,6 +295,9 @@ async function submissionActionNotification(userInfo, action, aSubmission, userS
         case ACTIONS.CANCEL:
             await sendEmails.cancelSubmission(userInfo, aSubmission, userService, organizationService, notificationService);
             break;
+        case ACTIONS.RESUME:
```

Comment:
I know we discussed this in the backend developers meeting this morning, but I thought that we determined that we did not need an email notification for resuming a submission. If that's the case can this case be removed?